src/functions/cancellations-data-endpoint/testSiriSx.ts

```diff
@@ -0,0 +1,203 @@
+export const testSiri = `<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
```suggestion
export const testSiriSx = `<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```
And corresponding variables in the file. It's probs worth renaming the ones in `src/functions/avl-data-endpoint/testSiriVm.ts` to `*Vm` so that they both use more explicit names